Formula/cfn-lint.rb

```diff
@@ -3,8 +3,8 @@ class CfnLint < Formula
 
   desc "Validate CloudFormation templates against the CloudFormation spec"
   homepage "https://github.com/aws-cloudformation/cfn-python-lint/"
-  url "https://github.com/aws-cloudformation/cfn-python-lint/archive/v0.22.3.tar.gz"
-  sha256 "d921e54cb599edec87381462f6c1e700acf342f5fd421d22afe31784c5f281b0"
+  url "https://github.com/awslabs/cfn-python-lint/archive/v0.22.4.tar.gz"
```

Comment:
Shouldn't the `url` be `https://github.com/aws-cloudformation/cfn-python-lint/archive/v0.22.4.tar.gz`?

Reply:
yeah, not sure why I used awslabs, but `https://github.com/awslabs/cfn-python-lint` would redirect to the `aws-cloudformation/cfn-python-lint` one.

Either way, I will switch that back to reduce the confusion.

Reply:
Updated